src/pages/es/guides/deploy.md

```diff
@@ -160,8 +160,8 @@ jobs:
 
 ## GitLab Pages
```

Comment:
(Since this is `dist` in the English page, I'm assuming that `build` was wrong before, and that this correction is necessary. Just pointing this out because it's otherwise the only non buildOptions->build change in this PR.)